### PR TITLE
Add needles for wine test

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

File: steam-login-startup-libgl_error-20180331.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "tags": [
+    "steam-startup-libgl_error"
```
Comment (asmorodskyi) [inline steam-login-startup-libgl_error-20180331.json]: Can't find usage of this tag in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4749

File: steam-agreement-ok-leap-20180401.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (asmorodskyi) [inline steam-agreement-ok-leap-20180401.json]: Why you need this needle ? Looks identical to steam-agreement-ok but with console in background which should be unrelated.

File: steam-login-startup-libgl_error-20180331.json (hunk 1 above)
Comment (okurz) [reply steam-login-startup-libgl_error-20180331.json]: See https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4748/files#diff-32797c94ffb120abddf9f3997212c570R29

File: steam-agreement-ok-leap-20180401.json (hunk 2 above)
Comment (okurz) [reply steam-agreement-ok-leap-20180401.json]: The PR includes two variants, for openSUSE Tumbleweed, no prefix in name, and openSUSE Leap, with "-leap-" in name

File: steam-login-startup-libgl_error-20180331.json (hunk 1 above)
Comment (asmorodskyi) [reply steam-login-startup-libgl_error-20180331.json]: ok

**Pull request merged**